db/migrations/0003_frontend_port_constraint.sql

```diff
@@ -0,0 +1,4 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE frontends ADD CONSTRAINT idx_frontends_port_tenant_id CHECK (port > 1 AND port < 65536);
```

Comment:
Should this be API validation instead of DB validation?

Reply:
I added some API layer validation, too.